src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2405,33 +2406,59 @@ private static byte[] ConvertEnvVarsToByteArray(StringDictionary sd)
 
         private void SetStartupInfo(ProcessStartInfo startinfo, ref ProcessNativeMethods.STARTUPINFO lpStartupInfo, ref int creationFlags)
         {
-            bool hasRedirection = false;
+            // If we are starting a process using the current console window, we need to set its standard handlers
+            // explicitly when they are not redirected because otherwise they won't be set.
+            //
+            // However, if we are starting a process with a new cosnole window, we should not explicitly set those
```

Comment:
```suggestion
            // However, if we are starting a process with a new console window, we should not explicitly set those
```

Such a minor nit but it nagged me when reviewing the code :)

Reply:
> because otherwise they won't be set and <what?>

Could you please add words about the consequences?

Reply:
Comment updated.